### PR TITLE
(chore) add shipwright

### PR DESCRIPTION
**Pull request by toddself**
use shipwright to build images

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+sudo: required
+language: node_js
+services:
+- docker
+os:
+- linux
+node_js:
+- '8'
```
Comment (zxaos) [inline .travis.yml:8]: Can we add 10 since we're going to have to upgrade within the year?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+git:
+  depth: false
+cache:
+  directories:
+  - ".git"
+before_install:
+- openssl aes-256-cbc -k ${GCR_KEY} -in key.json.enc -out key.json -d
+- npm i @npm-wharf/shipwright@latest -g
+- npx makeshift -s @npm -r https://registry.internal.npmjs.com
```
Comment (zxaos) [inline .travis.yml:18]: Do we need to npx this? It requires a whole separate install sequence to echo one line

File: package.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  "name": "npm-docker-elasticsearch",
+  "version": "1.0.0",
+  "description": "Elasticsearch with GCS Fuse",
+  "main": "Dockerfile",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/npm/npm-docker-elasticsearch.git"
```
Comment (zxaos) [inline package.json]: prefer plain git repo urls?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "npm-docker-elasticsearch",
+  "version": "1.0.0",
+  "description": "Elasticsearch with GCS Fuse",
+  "main": "Dockerfile",
```
Comment (zxaos) [inline package.json]: main isn't going to work here... does it make sense to even have one?

**Review by zxaos — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (toddself) [reply package.json]: this is what npm gave me when i ran `npm init`

File: .travis.yml (hunk 2 above)
Comment (toddself) [reply .travis.yml:18]: this travis yml is from the other repos -- just copied wholesale.  at least it's consistent?

File: package.json (hunk 3 above)
Comment (toddself) [reply package.json]: fixed it

**Review by zxaos — APPROVED** [review]

**Pull request merged**